### PR TITLE
[4.1] Scheduled Tasks Help

### PR DESCRIPTION
**Pull request by brianteeman**
This PR sets the up the help button correctly following the changes made by @bakual

This PR removes the help and options buttons from the select task page to be consistent with similar pages such as select new module

The language strings are now in place for the help table of contents administrator/index.php?option=com_admin&view=help

The help TOC cannot be generated yet to include Scheduled Tasks because of #36713

@tecpromotion sorry forgot the code had changed so you might need to rename the page you created and create a new one for the single task new/edit

### CODE REVIEW

File: administrator/language/en-GB/com_admin.ini [hunk 1]
```diff
@@ -97,6 +97,8 @@ COM_ADMIN_HELP_REDIRECTS:_LINKS="Redirect: Links"
 COM_ADMIN_HELP_REDIRECTS:_NEW_OR_EDIT="Redirect: Links - New/Edit"
 COM_ADMIN_HELP_SITE_GLOBAL_CONFIGURATION="Global Configuration"
 COM_ADMIN_HELP_SITE_SYSTEM_INFORMATION="System Information"
+COM_ADMIN_HELP_SCHEDULED_TASKS="Scheduled Tasks"
+COM_ADMIN_HELP_SCHEDULED_TASKS:_EDIT="Scheduled Tasks: New/Edit"
```
Comment (richard67) [inline administrator/language/en-GB/com_admin.ini:101]:
```suggestion
COM_ADMIN_HELP_SCHEDULED_TASKS_EDIT="Scheduled Tasks: New/Edit"
```

**Comment by tecpromotion** [comment]
@brianteeman sorry for the (very long) delay.
@coolwebcreations has created the appropriate pages and his team will fill them with content.

File: administrator/components/com_scheduler/src/View/Select/HtmlView.php [hunk 2]
```diff
@@ -135,14 +135,5 @@ protected function addToolbar(): void
 			->buttonClass('btn btn-danger')
 			->icon('icon-times')
 			->text(Text::_('JCANCEL'));
-
-		// Add preferences button if user has privileges
-		if ($canDo->get('core.admin') || $canDo->get('core.options'))
```
Comment (Quy) [inline administrator/components/com_scheduler/src/View/Select/HtmlView.php:140]: Remove lines 121-122.

File: administrator/components/com_scheduler/src/View/Select/HtmlView.php [hunk 3]
```diff
@@ -118,8 +118,6 @@ public function display($tpl = null): void
 	 */
 	protected function addToolbar(): void
 	{
-		$canDo = ContentHelper::getActions('com_scheduler');
```
Comment (Quy) [inline administrator/components/com_scheduler/src/View/Select/HtmlView.php:121]: Please remove line 17.

File: administrator/language/en-GB/com_admin.ini (hunk 1 above)
Comment (Quy) [inline administrator/language/en-GB/com_admin.ini]:
```suggestion
COM_ADMIN_HELP_SCHEDULED_TASKS="Scheduled Tasks"
COM_ADMIN_HELP_SCHEDULED_TASKS:_EDIT="Scheduled Tasks: New/Edit"
COM_ADMIN_HELP_SITE_GLOBAL_CONFIGURATION="Global Configuration"
COM_ADMIN_HELP_SITE_SYSTEM_INFORMATION="System Information"
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a6c20d7bb148c271a4b7c22d0fe67e8a45daed78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36714">issues.joomla.org/tracker/joomla-cms/36714</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on a6c20d7bb148c271a4b7c22d0fe67e8a45daed78<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36714">issues.joomla.org/tracker/joomla-cms/36714</a>.</sub>

**Comment by jwaisner** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36714">issues.joomla.org/tracker/joomla-cms/36714</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thankyou

**Pull request merged**